modal.js

```diff
@@ -0,0 +1,26 @@
+var containerModal = document.querySelector(".modal");
+var btnModalOpen = document.querySelector(".btn-modal-open");
+var btnModalClose = document.querySelector(".btn-modal-close");
+
+containerModal.style.display = "none";
+console.log(containerModal)
+console.log(btnModalClose)
+console.log(btnModalOpen)
+var openModelHandler = () => {
+    containerModal.style.display = "flex";
+    btnModalOpen.style.display = "none";
+    console.log(true)
+}
+
+var closeModelHandler = () => {
```

Comment:
use let and const where possible, like this function can be declared with const keyword.

Reply:
fixed